src/tests/topp/MzTabExporter_5_output.mzTab

```diff
@@ -0,0 +1,145 @@
+MTD	mzTab-version	1.0.0
+MTD	mzTab-mode	Summary
+MTD	mzTab-type	Quantification
+MTD	title	ConsensusMap export from OpenMS
+MTD	description	OpenMS export from consensusXML
+MTD	protein_search_engine_score[1]	[, , Epifany Posterior Probability, 2.6.0-pre-feature-cXMLProtGroups-2020-04-06]
+MTD	peptide_search_engine_score[1]	[, , Percolator Posterior Error Probability, 3.02]
+MTD	peptide_search_engine_score[2]	[, , XTandem, ]
+MTD	psm_search_engine_score[1]	[, , Percolator Posterior Error Probability, 3.02]
```

Comment:
different number of peptide and psm search engine scores...

Reply:
Yes, which is now correct. In theory we should be able to conserve the underlying search engine score. It is not really implemented yet since you need a big map of possible score types for each search engine and do a pre lookup through all the peptide hits meta values.